swagger.json

```diff
@@ -0,0 +1,1344 @@
+        },
+        "unit": {
+          "type": "string",
+          "x-go-name": "Unit"
+        },
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "Plugin": {
+      "type": "object",
+      "title": "Plugin represents a plugin type definition.",
+      "properties": {
+        "config_policy": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyTable"
+          },
+          "x-go-name": "ConfigPolicy"
+        },
+        "hitcount": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "HitCount"
+        },
+        "href": {
+          "type": "string",
+          "x-go-name": "Href"
+        },
+        "id": {
+          "type": "integer",
+          "format": "uint32",
+          "x-go-name": "ID"
+        },
+        "last_hit_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LastHitTimestamp"
+        },
+        "loaded_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LoadedTimestamp"
+        },
+        "name": {
+          "type": "string",
+          "x-go-name": "Name"
+        },
+        "pprof_port": {
+          "type": "string",
+          "x-go-name": "PprofPort"
+        },
+        "signed": {
+          "type": "boolean",
+          "x-go-name": "Signed"
+        },
+        "status": {
+          "type": "string",
+          "x-go-name": "Status"
+        },
+        "type": {
+          "type": "string",
+          "x-go-name": "Type"
+        },
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "PolicyTable": {
+      "$ref": "#/definitions/RuleTable"
+    },
+    "PolicyTableSlice": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/RuleTable"
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "ProcessWorkflowMapNode": {
+      "type": "object",
+      "required": [
+        "plugin_name"
+      ],
+      "properties": {
+        "config": {
+          "description": "Config the configuration of a processor.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "x-go-name": "Config"
+        },
+        "plugin_name": {
+          "type": "string",
+          "x-go-name": "PluginName"
+        },
+        "plugin_version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "PluginVersion"
+        },
+        "process": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProcessWorkflowMapNode"
```

Comment:
I think we should wait for the next release.  The issue you referenced looks promising in that they appear to be addressing it.